source/ios/AdaptiveCards/ADCIOSVisualizer/resources/sample.json

```diff
@@ -100,7 +102,7 @@
     },
     "imageSet": {
         "imageSize": "Medium",
-        "maxImageHeight": "maxImageHeight"
```

Comment:
This is a test break introduced from outside. Giving an explicit int number to work around.